### PR TITLE
Minor improvements and extension of picture tag

### PR DESCRIPTION
**Pull request by jwfwessels**
updated template markup, added object fit variable, minor tweaks and improvements to picture tag

### CODE REVIEW

File: responsive_image/templatetags/responsive_image_tags.py [hunk 1]
```diff
@@ -85,9 +92,15 @@ def render(self, context):
...
             name, extension = canonical_url.rsplit(".", 1)
 
-            for di, size_name in zip(srcsets, size_names.split(",")):
-                url = getattr(obj, "get_%s_url" % size_name)()
-                di["url"] = url
+            try:
+                for di, size_name in zip(srcsets, size_names.split(",")):
+                    url = getattr(obj, "get_%s_url" % size_name)()
+                    di["url"] = url
+            except AttributeError:
```
Comment (hedleyroos) [inline responsive_image/templatetags/responsive_image_tags.py]: It's better to do a `if hasattr` check earlier instead of relying on exceptions.

Comment (jwfwessels) [reply responsive_image/templatetags/responsive_image_tags.py]: hm ok cool. I also didn't like this solution but couldn't find any info online as to a better approach will investigate thanks :)

**Comment by hedleyroos** [comment]
It is missing documentation (what is object_fit?) and there  are no tests for the new variable. Changelog and Readme needs updating.

**Comment by hedleyroos** [comment]
I think this PR must be deleted because of the large changes in develop after our discussion @jwfwessels.

**Pull request closed without merging**